types/advocates.ts

```diff
@@ -25,4 +25,19 @@ interface Advocate {
   slackUsername?: string;
 }
 
-export { AdvocatesWorldRegion, ADVOCATES_WORLD_REGION_OPTIONS, Advocate };
+interface QampProject {
+  name: string;
+  url: string;
+  website?: string;
```

Comment:
I believe we're only display one URL which is the GitHub issue, right?

If that's the case, I think that `githubIssueUrl: string` might be more precise.